common/dev/mp2971.c

```diff
@@ -919,6 +919,18 @@ bool mp2971_vid_to_direct(sensor_cfg *cfg, uint8_t rail, uint16_t *millivolt)
 	CHECK_NULL_ARG_WITH_RETURN(cfg, false);
 	CHECK_NULL_ARG_WITH_RETURN(millivolt, false);
 
+	bool vout_scale_enable = false;
+	if (cfg->init_args != NULL) {
+		mp2971_init_arg *init_arg = (mp2971_init_arg *)cfg->init_args;
+		vout_scale_enable = init_arg->vout_scale_enable;
+	}
+	float vout_scale = 1.0;
+	if (vout_scale_enable == true) {
```

Comment:
why is this different than the other spots? The only way vout_scale_enable can be true is in the previous `if` so why not add this block to the previous `if`? 

Also style nit: drop the `== true`